### PR TITLE
Add new powershell.codeFormatting settings for new options in PSSA 1.18: WhitespaceInsideBrace and WhitespaceAroundPipe

### PR DESCRIPTION
**Pull request by bergmeister**
## PR Summary

This PR exposes new settings to use the new PSSA settings from this [PR](https://github.com/PowerShell/PSScriptAnalyzer/pull/1092).
The corresponding PR in PSES is [here](https://github.com/PowerShell/PowerShellEditorServices/pull/835)
This PR can only be merged once PSSA 1.18 has released and the dependent PSES has been merged.

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- [ ] PR has tests
- [ ] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

**Comment by bergmeister** [comment]
The depending PSSA PR has been merged now (meaning the feature will be in 1.18), so only the release is blocking the PR from merging, can you please start the review @TylerLeonhardt or @rjmholt in the meantime?

**Comment by rjmholt** [comment]
(Taken the merge note out, since we won't release before PSSA does)

File: package.json [hunk 1]
```diff
@@ -611,6 +611,16 @@
           "default": true,
           "description": "Adds a space after a separator (',' and ';')."
         },
+        "powershell.codeFormatting.WhitespaceInsideBrace": {
+          "type": "boolean",
+          "default": true,
+          "description": "Adds a space after an opening brace ('}') and before a closing brace ('}')."
```
Comment (rkeithhill) [inline package.json]: You say "opening brace" but then use a closing brace as an example.  I think that should be `('{')`.

File: package.json [hunk 2]
```diff
@@ -611,6 +611,16 @@
...
+        },
+        "powershell.codeFormatting.WhitespaceAroundPipe": {
+          "type": "boolean",
+          "default": true,
+          "description": "Adds a space before and after a pipeline ('|')."
```
Comment (rkeithhill) [inline package.json]: I think we should refer to this symbol as it is referred to in the docs "pipeline operator" e.g. "before and after the pipeline operator ('|').".

File: package.json (hunk 1 above)
Comment (bergmeister) [inline package.json]:
```suggestion
          "description": "Adds a space after an opening brace ('{') and before a closing brace ('}')."
```

File: package.json (hunk 2 above)
Comment (bergmeister) [inline package.json]:
```suggestion
          "description": "Adds a space before and after the pipeline operator ('|')."
```

File: package.json (hunk 1 above)
Comment (bergmeister) [reply package.json]: Well spotted, thanks. I fixed it

**Pull request merged**